app/scripts/modules/core/src/pipeline/manualExecution/manualPipelineExecution.html

```diff
@@ -7,7 +7,10 @@ <h3 ng-if="vm.command.pipeline && !vm.triggers.length">Confirm Execution</h3>
     <h3 ng-if="!vm.command.pipeline">Select Pipeline</h3>
   </div>
   <ng-form name="manualExecutionForm">
-    <div class="modal-body container-fluid form-horizontal">
+    <div ng-if="vm.planError" class="alert alert-danger">
```

Comment:
manual changes in this file are lines 10 through 12 and line 170. The rest was prettier formatting.